### PR TITLE
model boilerplate submission 1

### PR DESCRIPTION
**Pull request by tcliff30**
created a boilerplate for the Model and testing it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Python Automated Testing Example
```
Comment (sagars729) [inline README.md:1]: You don't need this README as part of this commit - you can remove this or modify it to fit your current project structure.

Comment (tcliff30) [reply README.md:1]: okay i'll fix that

File: test/test_model.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from src.Model import Model
+
+def test_model():
+  assert(Model.evaluate(Model)[0] == 0.4)
```
Comment (sagars729) [inline test/test_model.py:4]: You aren't guaranteed that the model will always evaluate the first action value to 0.4. I think you should instead add an assertion about the 
- shape of the result 
- the type of data
- or the bounds

You should also avoid passing in model to evaluate like that. Instead do,
```python
model = Model(image, height, width)
# Then use model.evaluate() with your assertions
```

File: test/test_model.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+from src.Model import Model
+
```
Comment (sagars729) [inline test/test_model.py:2]: Add a test for __init__

File: test/src/Model.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class Model:
+
+	def __init__(self, image, height, width):
+            self.image = image
+            self.height = height
+            self.width = width
+
+	def evaluate(self):
+            return [0.4,0.1,0.1,0.1,0.1,0.1,0.1]
```
Comment (sagars729) [inline test/src/Model.py:9]: Evaluate most likely needs an image (or multiple images) as an input

**Comment by sagars729** [comment]
Looks good to me - the tests and methods will need to be updated as this project progresses, but this looks good for a simple boilerplate!

**Review by sagars729 — APPROVED** [review]

**Pull request merged**